views/admin/docappt.php

```diff
@@ -123,18 +130,23 @@
            echo "<input type='text' name='doc' value=" . $id . ">";
            echo "<input type='submit' name='press' value='Submit'>";
            echo "</form>";
-        }
-        ?>
 
 
-      <label for="patients_name" class='display_only'>Patient Name:
-        <p class='display_only' ><?php if(isset($_POST['submit'])) echo $patient_name ?></p>
-      </label>
+           echo "<label for='patients_name' class='display_only'>Patient Name:";
+             echo "<p class='display_only' >" . $patient_name . "</p>";
+           echo "</label>";
+         }
+
+
+
+        }
+        ?>
 
       <?php
 
         if(isset($_POST['press'])) {
```

Comment:
It seems a little redundant to close the `<?php>` only to open it again right away.  Unless you were planning on adding html here I think it be better to combine them